### DISCUSSION TITLE
Set specific sbus values

### DISCUSSION THREAD

**Discussion by labomat** [EdgeTX/edgetx#3168] [Q&A]
Hello, I am looking for a way to send specific sbus values for one channel to the receiver. At best from touchscreen events and from external switches. My best idea at the moment is to use the trainer input for external input but this would not help with touch screen events.

Background: I am using EdgeTx with a model boat with a lot of special functionality (sound, light, different cranes etc.) and I am looking for a solution to build a multiswtich/multiprop solution. So I would like to send my own data.

Any suggestions are welcome, Kai

**Comment by rotorman** [comment]
Have a look if the LUA widget in https://github.com/Ziege-One/TSwitch might already be what you are looking for. Here a German language video showing it in action https://www.youtube.com/watch?v=V8RkckBJeKk
